pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -2,7 +2,17 @@
 
 with import ./lib.nix { inherit pkgs; };
 
-self: super: {
+self: super:
+
+let
+  # apply an override to a package if it’s build with GHC <8.0
+  ifLT8 = override: drv:
```

Comment:
I didn’t want to expose that (yet), since it’s not completely clear if it’s a helpful library function (at least for me).